BaseLib/IO/XmlIO/Qt/XMLQtInterface.h

```diff
@@ -40,17 +40,17 @@ class XMLQtInterface
     int isValid() const;
 
     /// Checks if a hash for the given data file exists to skip the time-consuming validation part.
-    /// If a hash file exists _and_ the hash of the data file is the same as the content of the hash file the validation is skipped
```

Comment:
Still, there are unrelated changes in the comments

Reply:
I only find this one.